packages/sdk/Dockerfile

```
@@ -169,5 +169,67 @@ ADD --chmod=644 \
 RUN tar -xJf /usr/share/cartesi-machine/images/linux-headers.tar.xz -C / && \
     rm /usr/share/cartesi-machine/images/linux-headers.tar.xz
 
+# Install migrate db cli
+ARG GO_MIGRATE_VERSION=4.18.2
```

```suggestion
ARG GO_MIGRATE_VERSION
```

This should just be declared.